### DISCUSSION TITLE
same prefix for rest and graphql

### DISCUSSION THREAD

**Discussion by aryala7** [rebing/graphql-laravel#870] [Q&A] [answered]
hey guys I have a little problem with my API routes. both rest apis and graphql apis uses the same prefix. so when I try to send a request to one of my rest method URLs it will return a blank page with an empty bracket but if I change the prefix; the problem will be solved. is it wrong to set the same prefix for API and graphql URLs?

<img width="821" alt="Screen Shot 2021-12-13 at 5 37 21 PM" src="https://user-images.githubusercontent.com/56627259/145826973-3e1792ef-a895-46a4-8432-09ed78abff64.png">
<img width="821" alt="Screen Shot 2021-12-13 at 5 37 40 PM" src="https://user-images.githubusercontent.com/56627259/145827013-5fe26d7d-e8f6-4b62-adef-eca615b7f3dc.png">

**Comment by mfn** [comment, marked as answer]
> is it wrong to set the same prefix for API and graphql URLs?

How would the clients otherwise specify which one to contact?

Ok maybe I missed something obvious 😅 but my first thought is: it has to be different, it's not the same. There needs to be a clear discriminator, since both may use `GET` and `POST` requests for example.

**Reply by aryala7** [reply]
thanks for your kind reply Markus. no you didnt miss anything :) have a great day
